hosts.json

```
@@ -0,0 +1,24 @@
+[
+	{
+		"url": "https://api.wordpress.org",
+		"label": "Wordpress.org",
```

```suggestion
		"label": "WordPress.org",
```
Oh, and this :joy: Let's not have to deal with those bug reports 😝